server/src/database/models/index.ts

```diff
@@ -2,16 +2,24 @@
 //
 // SPDX-License-Identifier: MIT
 
+import Attainable from './attainable';
 import Course from './course';
-import CourseAssignment from './courseAssignment';
 import CourseInstance from './courseInstance';
-import CourseInstancePartialGrade from './courseInstancePartialGrade';
 import CourseResult from './courseResult';
 import CourseRole from './courseRole';
 import CourseTranslation from './courseTranslation';
 import User from './user';
-import UserAssignmentGrade from './userAssignmentGrade';
-import UserPartialGrade from './userPartialGrade';
+import UserAttainableGrade from './userAttainableGrade';
+
+Attainable.hasMany(Attainable, {
```

Comment:
This is more of a general comment than this PR, but why are all the associations defined in this one file? Wouldn't it be better to define, e.g., `Course` associations in `models/course.ts`?

Reply:
I think the original idea was to clearly separate the models and sequelize associations. I think this worked well when we had few(er) associations, you could easily read and edit associations in one file. But now that the `index.ts` file is slowly growing bigger, maybe it would be better to have a look on this issue in the near future.